packages/playground/assets/__tests__/assets.spec.ts

```diff
@@ -304,6 +304,8 @@ if (!isBuild) {
       (code) => code.replace('#0088ff', '#00ff88'),
       true
     )
+    // wait for page reload
+    await timeout(200)
```

Comment:
```suggestion
    await page.waitForNavigation()
```
There's an function to wait for reload.